.github/ISSUE_TEMPLATE/02_bug.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: A concrete bug report with steps to reproduce the behavior.
+labels: bug,
+---
+
+### Bug description
+
+
+
+### How is the bug triggered?
+How can you reproduce the bug?
+1.
+
+
+### What did you expect to happen?
+
+
+
+### How are you running the application?
+Please include at least:
+- **Zulip-terminal version:**
+  eg. a specific version (0.7.0), or if running from `main` also ideally the git ref
+- **Zulip server version:**
```

Comment:
I noticed that you chose to drop the checkbox list for Zulip server versions.
I'm not quite acquainted with the server versions yet.
But, would the same checkbox not apply for ZT?

Reply:
See the 'parity' file, which links to the API changelog for some server versions.

I will add chat.zulip.org as one of the Zulip server suggested version options.

Reply:
We could add each server version, but we've historically aimed to support from 2.1 onwards. That doesn't mean we support everything that web has, but we've not dropped support for older server versions. For that reason the list of server versions to select from wouldn't be 8/7/6/5(orless), but more like 8/7/6/5/4/3/2.1. That's only two more options, but it might make the list a little long, and even longer with new releases ;)